packages/template-blank-svelte/package.json

```
@@ -13,11 +13,11 @@
   "files": [
     "app",
     "App_Resources",
+    "types",
     "hooks",
     "tools",
     "!tools/assets",
-    ".editorconfig",
-    "references.d.ts",
+    ".gitignore",
```

Don't need to add `.gitignore` because it's part of the `tools` folder, and handled by the `after-createProject` hook.

```suggestion
    ".editorconfig",
```